robot/robot.py

```diff
@@ -139,7 +139,7 @@ def _games(self) -> BoardList[GameState]:
         return self._update_boards(self.known_gamestates, GameState, 'game')
 
     @staticmethod
-    def _single_index(name, list_of_boards: BoardList[TBoard]) -> TBoard:
+    def _single_index(name: Any, list_of_boards: BoardList[TBoard]) -> TBoard:
```

Comment:
Is there something we can constrain it down to besides `Any`?

Reply:
huh, that should, I think, be `str`? Not sure why I made that `Any`.

Reply:
Ah, I know why I made that `Any` -- because that's the technically correct type. The only thing we do with the name is format it intio a string, so we don't care what type it is.

Reply:
I think we should still constrain it to a better type that reflects what it should be, even if the implementation doesn't actually care what type it is

Reply:
c393d18